client/my-sites/site-settings/composing/index.jsx

```diff
@@ -29,51 +30,54 @@ const Composing = ( {
 	setFieldValue,
 	siteIsJetpack,
 	updateFields,
-} ) => {
-	const CardComponent = siteIsJetpack ? CompactCard : Card;
+} ) => (
+	<>
```

Comment:
I often find that using `Fragment` instead of `<>` helps with readability. What do you think?

Reply:
I find the short syntax more intuitive because it's more aligned with what's actually rendered in the DOM (nothing) and because you don't need to import anything.

I see your point about readability, although I often take longer to process pieces of logic that contain `Fragment` components because at a first glance they look like a wrapping component which will render something until I realize they are just `Fragment`s.

This may be an interesting topic to bring up in the next Calypso DevEx meeting.

Reply:
> This may be an interesting topic to bring up in the next Calypso DevEx meeting.
Sure! Your rationale seems reasonable to me too! It seems it comes down to personal preference!